source/install.sh

```diff
@@ -4,7 +4,7 @@ SOURCE_PATH=$(dirname "$SCRIPT")
 
 # options
 # TODO: parse arguments and provide --help to set these options
-BUILD_TESTING="OFF"
+BUILD_TESTING="ON"
```

Comment:
😐 you probably want to leave this off, since normal users of the library don't need the tests. But it's a todo to make this options configurable from the command line so I guess I can't blame you for changing this here

Reply:
:D